src/reducers/tags.js

```diff
@@ -1,22 +1,30 @@
 import { TOGGLE_TAG_ACTIVE, TOGGLE_TAG_FILTER } from '../actions/tags';
 
+const activeChanges = action =>
+  action.tagIDs.reduce((result, tagID) => {
+    result[tagID] = action.active;
+    return result;
+  }, {});
+
+const enabledChanges = action =>
+  action.tagIDs.reduce((result, tagID) => {
+    result[tagID] = action.enabled;
+    return result;
+  }, {});
```

Comment:
You could abstract this very easily without significant changes:
```suggestion
const batchChanges = (action, key) =>
  action.tagIDs.reduce((result, tagID) => {
    result[tagID] = action[key];
    return result;
  }, {});
```

Reply:
Also this should probably have a small JSDoc comment? But no worries if you disagree

Reply:
Note for transparency: I've added a sneaky commit to patch this because @bru5 is away on PTO